<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreboardDialog</class>
 <widget class="QWidget" name="ScoreboardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scoreboard</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Weapon Performance Scoreboard</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="mDataDisplayTable">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total Fired</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Hit Count</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kill Count</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Miss Count</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>In Flight</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sorting Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,6,1">
      <item row="1" column="0">
       <widget class="QComboBox" name="mFilterComboBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Selection</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mSelectionComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="mHelperButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
